operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -298,6 +298,10 @@ checksum/config: {{ include (print .ctx.Template.BasePath "/mimir-config.yaml")
 {{ toYaml . }}
 {{- end }}
 {{- if .component }}
+{{- if .ctx.Values.useVaultAgent }}
+{{- $vaultAgentAnnotations := include "mimir.vaultAgent.annotations" .component }}
+{{- $vaultAgentAnnotations }}
```

Comment:
Is there any reason to assign the named template `mimir.vaultAgent.annotations` output to this variable?